docs/source/hardware/lowpriorityFAQ.rst

```diff
@@ -0,0 +1,259 @@
+partition are considered for execution on private partitions only when they are idle. 
+
+In other words, a low priority job will be executed only if the private partition has resources to execute a 
+job submitted to it. However, if a member of the private partition submits a job during this time, then the low 
+priority job will be automatically stopped/suspended and re-queued (preempted). 
+
+Jobs submitted to the ``lowpriority`` partition have the lowest priority of any class of jobs and are only considered 
+after all other normal/high priority jobs have been placed. Jobs submitted by private partition members to their own 
+partition always have higher priority. 
+
+**5. Will the** ``lowpriority`` **partition become the default partition?**
+
+No, the default partition will still be partition `short`. By default, jobs are not subject to preemption.
+
+**6. How do I submit a job to the** ``lowpriority`` **partition?**
+
+The ``lowpriority`` partition is a distinct partition that must be requested explicitly. SLURM accepts a 
+comma-separated list of partitions. If a job is scheduled to run in the ``lowpriority`` partition, it may be 
+preempted and re-queued. This does not happen in other partitions. The ``lowpriority`` partition is the only partition 
+that can be preempted. See examples `here <https://northeastern-university-rc-public-documentation--19.com.readthedocs.build/en/19/hardware/lowpriority.html>`_.
+
+**7. How can I ensure that my** ``lowpriority`` **partition job can run on nodes with sufficient resources?**
+
+It is important to specify the required resources in the job script. For CPUs, this can include the number of cores 
+and the micro-architecture and for GPUs this is typically handled by “class”. The GPU classes that are configured on 
+the HPC system include P100, V100, T4, and A100. See our documentation `here <https://rc-docs.northeastern.edu/en/latest/using-discovery/workingwithgpu.html#working-gpus>`_.
+
+Resources
+===================
+
+**1. What are the time limits on the** ``lowpriority`` **partition?**
+
+Partition limits for the ``lowpriority`` partition are the same as the default partition (short), which has a 4h 
+default walltime and a 24h max-walltime.
+
+**2. What are the resource and job limits on the** ``lowpriority`` **partition?**
+
+Currently, the resource limits and the number for CPU-supported low priority jobs on the ``lowpriority`` partition 
+are the same as the `default partition <https://rc-docs.northeastern.edu/en/latest/hardware/partitions.html>`_. The 
+resource limits and the number for GPU-supported low priority jobs on the ``lowpriority`` partition are the same as 
+the `gpu partition <https://rc-docs.northeastern.edu/en/latest/hardware/partitions.html>`_.
+
+**3. Is there a limit on the number of CPUs or GPUs that can be requested when trying to use the** ``lowpriority`` **partition?**
+
```

Comment:
there's GPUs in the question, but no mention of GPUs in the answer.

Reply:
I've added GPU to the answer